### ISSUE TITLE
Wrong ascension material specified to Lynette

### ISSUE THREAD

**Issue by arkni8** [MadeBaruna/paimon-moe#450]
The ascension material metioned in Paimon.moe for Lynette seems to be wrong. It seems the ascension material is not `Artificed Spare Clockwork Component — Coppelius`. From the image, you can see the ascension material is actually `Artificed Spare Clockwork Component — Coppelia`

![image](https://github.com/MadeBaruna/paimon-moe/assets/96630066/9f9ae191-173a-4b32-b9d0-3b50271a0341)

![image](https://github.com/MadeBaruna/paimon-moe/assets/96630066/69d7b76e-b982-4dd6-a44a-b0f0c1d99092)

**Comment by arkni8** [comment]
Apologies if this is not where people report such errors 😥

**Comment by Ilya0S** [comment]
I'd say it's more of a case of wrong name associated to material, not wrong material associated to character.

**Comment by phritzy** [comment]
I made a pr that I think should fix this #458.

**Comment by kittisusi** [comment]
Yeah I've noticed this too; the icon for coppelius (the white one used by Freminet) is incorrect, using instead the blue icon. And lynette is listed to use Coppelius in the calculator with the correct icon, but she actually uses Coppelia. Hence in the total count they are combined in count when they shouldn't be, due to using different materials. 
![Capture](https://github.com/MadeBaruna/paimon-moe/assets/77296416/6025b0ec-1838-4415-91fb-86a7053a14f0)
Freminet: Has correct name but incorrect icon
Lynette: Has correct icon but incorrect name

As a result they combine counts for the one item when each uses a different one.
![image](https://github.com/MadeBaruna/paimon-moe/assets/77296416/0ff607b6-7d67-4e56-a728-931a7a493ce7)
